src/parser/lexer/mod.rs

```diff
@@ -86,6 +86,8 @@ pub enum Command {
     SetScale,
     ShiftFrequency,
     ShiftPhase,
+    #[strum(to_string = "SWAP-PHASES")]
```

Comment:
Is this not already the string representation of `SwapPhases`, given the enum uses `SCREAMING-KEBAB-CASE` as the default?